Add a pre-made message about issue priority (#248)
meilisearch/integration-guides: resources/issues.md

We will accept and merge the first PR that fixes correctly and implements the issue following the contribution guidelines of the corresponding repository.

We are looking forward to your contributions. No need to wait for an assignment to start! 👍

## Issue priority feedback policy based on tier level

Hey \@username.
No reason why we shouldn't implement this, it's a good idea!

> I will add it to the stack of new features to add in this plugin, but we don't have enough time to work on it in the following weeks. 

Reviewer: Depending on the functionality:
"Unfortunately, the integration team won't have time to implement this improvement. I will create an issue so the community can easily implement it."
OR
"Unfortunately, the integration team won't have time to implement this improvement. I will add some details to this issue so the community can easily implement it."
WDYT?

Author: I think the first one does not work because the contributor already created the issue! But the last one I will use to improve the current message :D

> ## Issue priority feedback policy based on tier level
> 
> Hey \@username.
> I agree on implementing this! 
> 
> We might not have enough time to work on it in the following weeks.
> If someone wants to contribute and implement it, it will be released faster! PRs are always welcomed!

Author: ```suggestion
## Issue priority feedback policy based on tier level

Hey \@username.
I agree with implementing this! 

We might not have enough time to work on it in the following weeks. Also, I will add some details to this issue to make the implementation easier.
If someone wants to contribute and implement it, it will be released faster! PRs are always welcome!
```

> Hey \@username.
> No reason why we shouldn't implement this, it's a good idea!
> 
> I will add it to the stack of new features to add in this plugin, but we don't have enough time to work on it in the following weeks. 
> If someone wants to contribute and implement it, it can be faster! PRs are always welcomed!

Reviewer 2: ```suggestion
Hey \@username.
I agree on implementing this! 

We might not have enough time to work on it in the following weeks.
If someone wants to contribute and implement it, it will be released faster! PRs are always welcomed!
```
